src/lib/EntireScheme.svelte

```diff
@@ -89,13 +90,13 @@
     reader.readAsText(e.target.files[0]);
   }
 
-  function backfill(json) {
+  function backfill(json: Scheme) {
     let idCounter = 1;
     for (let f of json.features) {
       // Look for any LineStrings without length_meters. Old route-snapper versions didn't fill this out.
       if (f.geometry.type == "LineString" && !f.properties.length_meters) {
         f.properties.length_meters =
-          length(f.geometry, { units: "kilometers" }) * 1000.0;
+          length(f, { units: "kilometers" }) * 1000.0;
```

Comment:
Another case where the library is flexible in practice, but for some reason it doesn't type-check to do this. It makes more sense to pass in the whole feature anyway